### ISSUE TITLE
Android Studio doesn't understand SharedCode module in mpp-iOS-Android

### ISSUE THREAD

**Issue by r3h0** [JetBrains/kotlin-examples#96]
The mpp-iOS-Android doesn't quite work in the latest Android Studio. It builds because the Gradle part is set up properly, but the common code can't be edited inside Android Studio. Right now, I'd have to edit the Kotlin source in a basic text editor and build with Gradle terminal commands.

The attached screenshot shows how the `SharedCode_commonMain` doesn't show the common source files, resulting in the `createApplicationScreenMessage` function being unrecognized.

![image](https://user-images.githubusercontent.com/4955008/48227218-57dc6b80-e367-11e8-8663-b4a388166bba.png)

Steps to reproduce:

1. Clone kotlin-examples
2. Open root of mpp-iOS-Android project in the latest Android Studio (v3.2.1 as of now)
3. Note that the `SharedCode_commonMain` module doesn't have any source code listed
4. Open `app/src/main/java/com/jetbrains/jonnyzzz/myapplication/MainActivity.kt` and note that Android Studio doesn't know where the `createApplicationScreenMessage` function comes from, so it thinks it's an error

**Comment by r3h0** [comment]
@jonnyzzz, if there's anything I can help to investigate, let me know.

What I'm doing as a workaround for now is using the "Project" view in Android Studio, which lets me access the `commonMain` source code. After I build the project, navigating to `createApplicationScreenMessage` takes me to the decompiled source code. From there, I associate the decompiled code with the `commonMain` module. Since I'm running on Linux, I still get errors because of the iOS project; I had to "unload" those modules.

Shout out to @PeeJWeeJ for helping me find a workaround.

**Comment by jonnyzzz** [comment]
You may try to check that you have Kotlin plugin 1.3.0 in Android Studio. And please check you have Gradle (in `gradle.properties`) 4.7 or newer

**Comment by jonnyzzz** [comment]
It does not work for me too. Please follow the issue for Kotlin MPP. 
https://youtrack.jetbrains.com/issue/IDEA-202036

I was able to compile the project (in spite the errors in the editor), did you?

**Comment by r3h0** [comment]
That's correct; I can compile the project. The static analysis is limited, but the workaround I mentioned has been helpful.

**Comment by RafaelPlantard** [comment]
I'm facing this same issue, @jonnyzzz @r3h0 how did you guys fix it?

**Comment by RafaelPlantard** [comment]
Thanks @jonnyzzz 
I created the project using IntelliJ IDEA and after opened on Android Studio and it worked

**Comment by GoMino** [comment]
For my part, I just needed to add the dependency to the `SharedCode` module in the `build.gradle` of the `android project`.

```
dependencies {
    ...
    implementation project(':SharedCode')
    ...
}
```

Then sync your gradle, and your references should now resolve.

**Comment by egarc** [comment]
Misreading the above advice, I instead added `implementation project(':SharedCode')` to the project's `build.gradle` file. This created an error, so I removed that line, and gradle rebuilt along with the `bin` directory. Not sure what I inadvertently did, but if anybody has an explanation, this iOS dev would appreciate it 😄

**Comment by egarc** [comment]
I also ran into where the `xcode-frameworks` directory wasn't getting created. Running `./gradlew build` from the command line seemed to fix this.
